common/djangoapps/student/views.py

```diff
@@ -693,6 +703,47 @@ def _create_recent_enrollment_message(course_enrollment_pairs, course_modes):
         )
 
 
+def _create_credit_availability_message(username, enrolled_courses_dict, user):
+    """Builds a dict of credit availability for courses.
+
+    Construct a for courses user has completed and has not purchased credit
+    from the credit provider yet.
+
+    Args:
+        username (str): Username of the user.
+        course_enrollment_pairs (list): A list of tuples containing courses, and the associated enrollment information.
```

Comment:
Why you are sending username and user in this method. User object contains username also.